### ISSUE TITLE
Screen reader does notify user of successful project creation from cookiecutter template

### ISSUE THREAD

**Issue by eclectir** [Microsoft/PTVS#2242]
User also cannot tab to the status above the "open project in solution explorer" button.

**Comment by zooba** [comment]
There's no reason to tab to this status - we need to know how to expose that text to screenreaders without breaking keyboard navigation.

**Comment by eclectir** [comment]
Cool. The goal is for the screenreader to read this out at successful creation.

Get Outlook for iOS<https://aka.ms/o0ukef>




On Tue, Mar 7, 2017 at 5:19 PM -0800, "Steve Dower" <notifications@github.com<mailto:notifications@github.com>> wrote:


There's no reason to tab to this status - we need to know how to expose that text to screenreaders without breaking keyboard navigation.

**Comment by zooba** [comment]
Need further information on the right way to do this, so I'm not going to try and figure it out until we get that info.
